### PR TITLE
Fix README.md

### PR DESCRIPTION
**Pull request by ogapants**
Thank you for a wonderful tool!!
I fixed README to make it easier to understand usage 🤓

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,19 @@
 # slack_emoji_uploader_go
-Upload emoji file to Slack team easily - just drug and drop picture file.
+Upload emoji file to Slack team easily - **just drug and drop** picture file.
```
Comment (ogapants) [inline README.md:2]: Let's emphasize because it is a very wonderful feature 😃

File: README.md [hunk 2]
```diff
@@ -1,22 +1,19 @@
...
 
-##What is this script for?
+## What is this script for?
```
Comment (ogapants) [inline README.md:4]: fix markdown...

File: README.md [hunk 3]
```diff
@@ -1,22 +1,19 @@
...
-3. drug and drop picture file (currently jpg or png) to  
-   For Mac:  
-     upload.app  
-   For Windows:  
-     upload.bat  
-   (emoji name will be a file name without extension.)
-4. wait until emoji data are uploaded
-5. done!
+1. write all information to config file
+1. rename picture file (uploaded emoji name will be a file name without extension.)
```
Comment (ogapants) [inline README.md:14]: This is always necessary operation every time... 💭

**Comment by from-unknown** [comment]
Thank you for fixing README.
Honestly, current README is very poor so it is very helpful to enhance it.
I'll check your PR closer tonight.

**Comment by from-unknown** [comment]
Looks good to me.
Thank you for contribution!

**Pull request merged**